src/index.js

```diff
@@ -0,0 +1,168 @@
+import {
+	DateTimePicker,
+	RadioControl,
+	SelectControl,
+} from '@wordpress/components';
+import { store as coreStore, useEntityProp } from '@wordpress/core-data';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { addFilter } from '@wordpress/hooks';
+import { decodeEntities } from '@wordpress/html-entities';
+import { __ } from '@wordpress/i18n';
+
+const setAlertLevel = ( OriginalComponent ) => {
+	return ( props ) => {
+		const { slug } = props;
+
+		// Only override the alert level interface.
+		if ( 'alert_level' !== slug ) {
+			return <OriginalComponent { ...props } />;
+		}
+
+		const { editEntityRecord } = useDispatch( 'core' );
```

Comment:
If you scoot this down below the `useSelect()` chunk, you _could_ use
```js
const [ alertLevels, setAlertLevels ] = useEntityProp( 'postType', postType, slug );
```

`alertLevels` would then take care of `selectedTerms`, and `setAlertLevels( [ termID ] );` would take care of `editEntityRecord( 'postType', postType, postId, data );`

Reply:
🥇 good call! This had a great cascading effect in 087fb1d2a8c078a906458d8ede524cbe7bf78eb2